helpers/ktx-preview.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# ktx preview window helper
+
+set -eu
+
+selected=$1
+
+IFS=';' read -ra context <<< "$(kubectl config view -o jsonpath="{range .contexts[?(@.name == '${selected}')].context}{@.cluster};{@.user};{@.namespace};{end}")"
+cluster=${context[0]}
+user=${context[1]}
+namespace=${context[2]}
+
+server=$(kubectl config view -o jsonpath="{.clusters[?(@.name == '${cluster}')].cluster.server}")
+
+echo "=== ${selected} ===\n"
+echo "Cluster ${cluster};Server ${server};User ${user};Namespace ${namespace};" | tr ';' "\n" | column -t
```

Comment:
Make sure to add a newline at the end of the file.

Reply:
Sure, will do. May I ask why?

Reply:
It's part legacy and part ease of use for other developers (using bash for example).
I think these articles best describe it:
http://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/
https://robots.thoughtbot.com/no-newline-at-end-of-file

Reply:
Today I learnt something new! Thanks for that, this somehow slipped my attention for all these years. Cool.